tests/ui/repr/16-bit-repr-c-enum.rs

```diff
@@ -0,0 +1,27 @@
+// build-pass
+// revisions: avr msp430
+//
+// [avr] needs-llvm-components: avr
+// [avr] compile-flags: --target=avr-unknown-gnu-atmega328 --crate-type=rlib
+// [msp430] needs-llvm-components: msp430
+// [msp430] compile-flags: --target=msp430-none-elf --crate-type=rlib
+#![feature(no_core, lang_items)]
+#![no_std]
+#![no_core]
```

Comment:
Why do we need this? To prevent this test from failing when `core` can't be compiled for these targets?

Reply:
Honestly I was just getting mysterious compilation errors while trying to build `core`, yeah.